code/game/machinery/doors/airlock.dm

```diff
@@ -1056,14 +1134,19 @@ There are 9 wires.
 	health -= crush_damage
 	healthcheck()
 
-
 /obj/structure/closet/airlock_crush(var/crush_damage)
 	..()
 	damage(crush_damage)
 	for(var/atom/movable/AM in src)
 		AM.airlock_crush()
 	return 1
 
+/obj/item/weapon/tool/airlock_crush(crush_damage)
+	. = ..() // Perhaps some function to this was planned, however currently this proc's return is not used anywhere, how peculiar. ~Luduk
+	unreliability += crush_damage * degradation * (1 - get_tool_quality(QUALITY_PRYING) * 0.01)
```

Comment:
In practise, the rate of damage seems way too high, throw in a *0.4 multiplier in here ontop of the other calculations